examples/plot/app.js

```diff
@@ -2,7 +2,7 @@
 import React, {Component} from 'react';
 import {render} from 'react-dom';
 import DeckGLOverlay from './deckgl-overlay.js';
-import {OrbitController} from 'deck.gl/dist/controllers';
+import {OrbitController} from 'deck.gl/dist-es6/controllers';
```

Comment:
This will not work for older browsers.
Instead, we need to make all controllers import `Viewport` from `deck.gl` instead of a relative path.